src/main/java/com/zendesk/maxwell/MaxwellReplicator.java

```diff
@@ -352,6 +385,14 @@ public RowMap getRow() throws Exception {
 						setReplicatorPosition((AbstractBinlogEventV4) v4Event);
 					}
 					break;
+				case MySQLConstants.ROTATE_EVENT:
```

Comment:
yeah, I'm mixing terms a *little* bit.  open-replicator's eof-mode stops at the end of all the data, whereas maxwell's stops after each file. 

I'm stopping after each file so that I can scan backwards file by file to find the heartbeat.  I start with the most recent binlog and walk backwards from there.